### DISCUSSION TITLE
How can we logout after login ?

### DISCUSSION THREAD

**Discussion by HelloReboot** [loco-rs/loco#713] [Q&A]
in the section of auth. we can login but no logout how can we logout and remove the token please ?

**Comment by jondot** [comment]
Hi, if you're meaning the JWT token, then this is guided by how JWT token are often used. They are often stored _client side_ after log in. And then, logging out is simply clearing them from client side store.
Does that make sense?

**Reply by HelloReboot** [reply]
think you for your reply bro. I got it.
